source/telemetry.txt

```diff
@@ -0,0 +1,125 @@
+.. _telemetry:
+
+===================
+Configure Telemetry
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+{+atlas-cli+}'s telemetry collects anonymous, aggregate usage data to
+identify improvements with the greatest impact. The 
+{+atlas-cli+} enables telemetry by default.
+
+Learn What the {+atlas-cli+} Tracks
+-----------------------------------
+
+{+atlas-cli+} telemetry tracks 
+non-Personally-Identifiable Information (PII), which includes but is
+not limited to the following information:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 60 40
+
+   * - Data
+     - Example Value
+
+   * - {+atlas-cli+} version number
+     - ``1.0.0``
+  
+   * - Installation source
+     - ``Homebrew``
+
+   * - Operating System (OS) and OS version
+     - ``Windows 11.5``
+
+   * - Authentication method. {+atlas-cli+} telemetry does *not* track
+       the values for the |api| keys and login credentials.
+     - ``API key``   
+
+   * - Details for commands you run. {+atlas-cli+} telemetry tracks 
+       arguments only if they use pre-set allowable values, such as 
```

Comment:
we don't track values only which flags where sent, in your example it would be `[ "provider", "region" ]`